lib/core/ui/root_widget.dart

```diff
@@ -63,6 +69,82 @@ Widget injectUMEWidget({
   );
 }
 
+class UMEWidget extends StatefulWidget {
+  const UMEWidget({
+    Key? key,
+    required this.child,
+    this.enable = true,
+    this.supportedLocales,
+    this.localizationsDelegates = defaultLocalizationsDelegates,
+  }) : super(key: key);
+
+  final Widget child;
+  final bool enable;
+  final Iterable<Locale>? supportedLocales;
+  final Iterable<LocalizationsDelegate> localizationsDelegates;
+
+  @override
+  _UMEWidgetState createState() => _UMEWidgetState();
+}
+
+class _UMEWidgetState extends State<UMEWidget> {
+  late Widget _child;
+
+  @override
+  void initState() {
+    super.initState();
+    _replaceChild();
+    _injectOverlay();
+  }
+
+  @override
+  void didUpdateWidget(UMEWidget oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if (widget.enabled != oldWidget.enabled && widget.enabled) {
```

Comment:
It should be `enable` ?

Reply:
Fixed. We might need an analyze test for CIs too.